content/docs/concepts/architecture/index.md

```diff
@@ -12,27 +12,28 @@ Keptn is an event-based control plane for continuous delivery and automated oper
 This event-driven architecture means that Keptn is tool and vendor agnostic.
 See [Keptn and other tools](../keptn-tools) for a fuller discussion.
 
-## Prerequisites
+## NATS
 
-During the installation of Keptn, [NATS](https://nats.io/) is installed into the Kubernetes namespace
+During Keptn installation, [NATS](https://nats.io/) is installed into the Kubernetes namespace
 where the Keptn Control Plane is installed.
+NATS is used to communicate with the Execution Plane as discussed below.
 
 ## Keptn CLI
 
-Use the [Keptn CLI](../../0.18.x/reference/cli/) to send commands
-that interact with the [Keptn API](../../0.18.x/reference/api/).
+Use the [Keptn CLI](../../0.19.x/reference/cli/) to send commands
+that interact with the [Keptn API](../../0.19.x/reference/api/).
 It must be [installed](../../install/cli-install)
 on the local machine and is used to send commands to Keptn.
 To communicate with Keptn, you need to know the API token
-which is managed as a shared secret by the [Secret Service](../../0.18.x/operate/secrets).
+which is managed as a shared secret by the [Secret Service](../../0.19.x/operate/secrets).
```

Comment:
The secret containing the API token (`keptn-api-token`) is technically not managed by the Secret service, but is created once during the installation via Helm

Reply:
done